frontend/container/etc/nginx/conf.d/default.conf

```diff
@@ -0,0 +1,30 @@
+server {
+    listen       80;
+
+    gzip         on;
+    gzip_vary    on;
+    gzip_types   application/javascript text/css;
```

Comment:
Is there a reason to restrict this to those types?

Reply:
Good point, should be fine to zip every thing if it is performant using the wildcard. text/html seems to be always compressed.